docs/operations.md

```diff
@@ -285,4 +451,4 @@ cluster is available in case of a disaster on the primary cluster.
 
 If you are using this strategy, we recommend you only support manual fail over from primary to secondary as
 automated strategies risk causing a [split brain](http://en.wikipedia.org/wiki/Split-brain_%28computing%29)
-problem.
+problem.
```

Comment:
keep the newline at the end of the file please (https://stackoverflow.com/a/729795/866066)